src/App.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {
+  BrowserRouter as Router, Routes, Route, Navigate,
+} from 'react-router-dom';
+import Layout from './components/Layout';
+import Boats from './components/pages/Boats';
+import BoatDetails from './components/pages/BoatDetails';
+import MyReservations from './components/pages/MyReservations';
+import Login from './components/pages/Login';
+import Signup from './components/pages/Signup';
+import Reserve from './components/pages/Reserve';
+import DeleteBoat from './components/pages/DeleteBoat';
+import AddBoat from './components/pages/AddBoat';
+
+const App = () => (
```

Comment:
- [ ] I can't reserve boats:

![image](https://github.com/Felipe-Perez-Ferraro/final_capstone_front/assets/57421823/b72f5c11-42aa-46ee-8802-eacd65058df5)

![image](https://github.com/Felipe-Perez-Ferraro/final_capstone_front/assets/57421823/9741da43-7da2-41cf-9ad7-38111954f16e)

![image](https://github.com/Felipe-Perez-Ferraro/final_capstone_front/assets/57421823/08f68a39-acea-4db7-b129-8978fac37f80)


Kindly, make sure boats can be reserved correctly so your app works as intended. 👍